src/work-allocation/components/task-assignment/task-assignment.component.spec.ts

```diff
@@ -2,6 +2,7 @@ import { Component, Input, ViewChild } from '@angular/core';
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 import { Router } from '@angular/router';
 import { Observable } from 'rxjs';
+import { SessionStorageService } from 'src/app/services';
```

Comment:
This import shouldn't begin with `src/`.

Reply:
Will change